main.tf

```diff
@@ -52,6 +28,12 @@ resource "aws_instance" "dp_web" {
   subnet_id                   = "${aws_subnet.data_pipe_apps.id}"
   private_ip                  = "${var.dp_web_private_ip}"
 
+  user_data = <<EOF
```

Comment:
does the machine have access to those parameters?

Reply:
i'll push that in a mo